uber/templates/preregistration/form.html

```diff
@@ -28,7 +28,7 @@ <h2 class="text-center">Dealer Application</h2>
           </div>
         {% endif %}
 
-        {% if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.BEFORE_DEALER_REG_SHUTDOWN %}
+        {% if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.DEALER_REG_OPEN %}
```

Comment:
`DEALER_REG_OPEN` is declared in `magprime`, correct? It shouldn't be referenced from the main `ubersystem` plugin if it's declared somewhere else.

I think you have to either move `DEALER_REG_OPEN` from `magprime` into `ubersystem`, or override this template in `magprime`.

Given that this is the prereg form, it'd probably be easier to move `DEALER_REG_OPEN` into `ubersystem`.

Reply:
`DEALER_REG_OPEN` is already in the uber plugin, we just define it again in magprime to override its behavior. Note that [here](https://github.com/magfest/ubersystem/pull/3259/files#diff-9c3818f655c258e1b711348de8f069c9R183), we already rely on it to override the form with a static page, so it definitely works. :)

Reply:
Oh yeah! You are totally right! I'll go approve the `magprime` pull request, too!